Makefile

```diff
@@ -104,10 +105,17 @@ third-party-test-venv: FORCE
 third-party-chpldoc-venv: FORCE
 	cd third-party && $(MAKE) chpldoc-venv
 
+test-venv: third-party-test-venv
+
 chpldoc: compiler third-party-chpldoc-venv
 	cd compiler && $(MAKE) chpldoc
 	@test -r Makefile.devel && $(MAKE) man-chpldoc || echo ""
 
+always-build-test-venv: FORCE
```

Comment:
Maybe it'd be smarter to make this part of the third-party-try-opt. i.e. always try to build it instead of only if some env var is set on the argument that end users will want to be able to run `make check`, though I suppose make check could simply rely on test-venv